pkg/graveler/graveler.go

```diff
@@ -979,7 +979,7 @@ func (g *KVGraveler) CreateBranch(ctx context.Context, repository *RepositoryRec
 		RunID:            postRunID,
 		StorageNamespace: storageNamespace,
 		EventType:        EventTypePostCreateBranch,
-		SourceRef:        ref,
+		SourceRef:        newBranch.CommitID.Ref(),
```

Comment:
Just verify - `newBranch` will always resolve to a commit ID?

Reply:
Reverting back. We can load the actions from the source ref, the actions filtering is using the new branch ID.